crucible/src/Lang/Crucible/CFG/SSAConversion.hs

```diff
@@ -273,6 +278,20 @@ extSwitchInfo :: SwitchInfo s blocks tp -> SwitchInfo s (blocks::>args) tp
 extSwitchInfo (SwitchInfo b typs a) = SwitchInfo (C.extendBlockID b) typs a
 #endif
 
+extBlockInfo
+  :: BlockInfo ext s ret blocks
+  -> BlockInput ext s (blocks ::> args) ret args
+  -> BlockInfo ext s ret (blocks ::> args)
+extBlockInfo bi binput = do
+  let blocks' = extBlockInputAssignment $ biBlocks bi
+  let jump_info' = extJumpInfoMap $ biJumpInfo bi
+  let switch_info' = extSwitchInfoMap $ biSwitchInfo bi
+  let breakpoints' = Bimap.mapR (mapSome C.extendBlockID) $ biBreakpoints bi
```

Comment:
I think this will reconstruct the breakpoint `Bimap` at every statement.  Is this a case where we can `unsafeCoerce` the whole map when `UNSAFE_OPS` is enabled?